CRM/Twingle/Submission.php

```diff
@@ -535,8 +541,9 @@ public static function createLineItems($values, $submission, $profile): array {
         $line_item_data['price_field_id'] = $price_field->price_field_id;
         $line_item_data['description'] = $price_field->description;
       }
-      // If the product ID is -22, it's an additional donation. See #102
-      elseif ($product['id'] === -22) {
+      // Identify additional donations by their ID.
+      // See https://github.com/systopia/de.systopia.twingle/issues/102
```

Comment:
I guess the link is not needed here again when it's documenting the constant above already.